src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -35,10 +35,9 @@ export const WithNavigationBar: React.FC<WithNavigationBarProps> = ({
   withHiddenLoginButton,
   children,
 }) => {
-  // @debt remove useVenueId from here and just pass it through as a prop/similar
-  const venueId = useVenueId();
-
-  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
+  const spaceSlug = useSpaceParams();
+  const { space } = useSpaceBySlug(spaceSlug);
+  const venueId = space?.id;
```

Comment:
If we're trying to get rid of `venue` word mentions we might as well call it `spaceId` everywhere similar constants are created, right?